hack/postrelease/test_manifests_archives.py

```diff
@@ -0,0 +1,54 @@
+import requests
+import tarfile
+
+from versions import RELEASE_VERSION
+
+urls = {
+    "docs": "https://github.com/projectcalico/calico/releases/download/{v}/release-{v}.tgz".format(
+        v=RELEASE_VERSION
+    ),
+    "github": "https://github.com/projectcalico/calico/releases/download/{v}/release-{v}.tgz".format(
+        v=RELEASE_VERSION
+    ),
+}
+
+manifest_list = [
+    "calico.yaml",
+    "calico-etcd.yaml",
+    "calico-bpf.yaml",
+    "calico-typha.yaml",
+    "calico-vxlan.yaml",
+    "calico-windows-bgp.yaml",
+    "calico-windows-vxlan.yaml",
+    "calicoctl.yaml",
+    "calicoctl-etcd.yaml",
+    "canal.yaml",
+    "canal-etcd.yaml",
+    "tigera-operator.yaml",
+    "custom-resources.yaml",
+]
+
+
+def test_manifest_archive_on_release_page():
```

Comment:
This works fine as-is, how hard would it be to make this a parameterised test (like in `test_tags_tars`).  That way the output is more explicit about exactly which files have been found/not found in the archive.